test/cases/ostree-raw-image.sh

```diff
@@ -478,6 +480,14 @@ enabled = ["custom.service"]
 EOF
 fi
 
+if [[ "${CUSTOM_FS_LVS}" == "true" ]]; then
+    tee -a "$BLUEPRINT_FILE" > /dev/null << EOF
+[[customizations.filesystem]]
+mountpoint = "/var/myfiles"
+size= "1 GiB"
```

Comment:
add some tests in root `/foo` , `/foo/bar`

Reply:
The suggested workaround has been added.